dataactvalidator/config/sqlrules/a34_appropriations.sql

```diff
@@ -0,0 +1,12 @@
+SELECT
+    approp.row_number,
+    approp.budget_authority_unobligat_fyb,
+    sf.amount as sf_133_amount
```

Comment:
I agree with the latter statement. Just because it's how it's been done previously doesn't mean it's still a good solution.